### DISCUSSION TITLE
.NET 5.0.404 SDK gives System.Core error in macOS.

### DISCUSSION THREAD

**Discussion by ayankumaar** [dotnet/AspNetCore.Docs#25656] [General]
I have created a nuget package. In that I have a .target file. In the target file under a task I am doing 

**<Reference Include="System.Core".** 

When I add the nuget package in my code and run a **dotnet build** command, I am getting the following error:

**error MSB3755: Could not find reference "System.Core". If this reference is required by your code, you may get compilation errors.**

I am just wondering is System.Core supported in MAC ? If not, what is the alternative I have ? 

If it is present in MAC then is it loaded from the GAC ? If yes, then what is the location of the GAC in MAC ?

**Comment by Rick-Anderson** [comment]
Why are you trying to use `<Reference Include="System.Core"` with .NET 5?
